src/int.js

```diff
@@ -213,6 +219,51 @@ Sk.builtin.int_ = Sk.abstr.buildNativeClass("int", {
             $doc: "the imaginary part of a complex number",
         },
     },
+    classmethods: {
+        from_bytes: {
+            $meth(args, kws) {
+                Sk.abstr.checkArgsLen("from_bytes", args, 0, 2);
+                let [bytes, byteorder, signed] = Sk.abstr.copyKeywordsToNamedArgs(
+                    "from_bytes",
+                    ["bytes", "byteorder", "signed"],
+                    args,
+                    kws,
+                    [Sk.builtin.bool.false$]
+                );
+                const s_lit = new Sk.builtin.str("little");
+                const s_big = new Sk.builtin.str("big");
+                if (byteorder !== s_big && byteorder !== s_lit) {
```

Comment:
This relies on interned strings in a way that freaks me out! Can I at least request a comment?

Reply:
Tweaked and extracted the logic so we don't rely on interning of strings here